### PR TITLE
libevent v2.1.12 (redux)

### PR DESCRIPTION
**Pull request by h-vetinari**
Saw that arrow wants to depend on this so I took an interest. Added myself to maintainers now that I saw there's a new version that hasn't been merged for a long time.

Also switch to cmake since it's a new version and arrow needs that metadata.

Closes #23

### CODE REVIEW

**Comment by h-vetinari** [comment]
@jakirkham, could you PTAL? I updated the feedstock to the latest version, added myself to maintainers, and switched the build to cmake (arrow needs the cmake config metadata).

**Comment by h-vetinari** [comment]
Gentle ping @jakirkham, could you PTAL? I'm willing to join here

**Comment by h-vetinari** [comment]
Another gentle ping @jakirkham :)

File: recipe/build.sh [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
-# Needed to ensure our OpenSSL and
-# not the system one is used on OS X.
-export LIBRARY_PATH="${PREFIX}/lib"
+mkdir build
+cd build
 
-export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
-export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
-export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+cmake -G "Ninja" \
```
Comment (xhochy) [inline recipe/build.sh]:
```suggestion
cmake -G "Ninja" ${CMAKE_ARGS} \
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -105,3 +94,4 @@ about:
 extra:
   recipe-maintainers:
     - jakirkham
+    - h-vetinari
```
Comment (xhochy) [inline recipe/meta.yaml:97]: Feel free to add me if this going to be an arrow dependency.
```suggestion
    - h-vetinari
    - xhochy
```

**Pull request merged**